fpr/forms.py

```diff
@@ -50,21 +49,12 @@ class Meta:
 class IDToolForm(forms.ModelForm):
     def clean(self):
         cleaned_data = super(IDToolForm, self).clean()
-        if self.instance.pk is None:
-            submitted_description = cleaned_data.get('description')
-            submitted_version = cleaned_data.get('version')
-
-            existing_idtool = get_object_or_None(
-                fprmodels.IDTool,
-                description=submitted_description,
-                version=submitted_version
-            )
-
-            if existing_idtool:
-                raise forms.ValidationError(
-                    _('An ID tool with this description and version already'
-                      ' exists'))
-
+        if self.instance.pk is None and fprmodels.IDTool.objects.filter(
+            description=cleaned_data.get('description'),
+            version=cleaned_data.get('version')).exists():
```

Comment:
Nitpick: I think you should add 4 spaces indentation before lines 53 and 54 (and pylint agrees with me).